src/stream/src/executor/source/reader.rs

```diff
@@ -67,13 +63,6 @@ impl SourceReaderStream {
         }
     }
 
-    #[stream(item = T)]
-    async fn paused_source<T>() {
-        yield futures::future::pending()
-            .stack_trace("source_paused")
```

Comment:
Can we keep the stack trace for paused source?

Reply:
Oops, I missed this point. It seems that we can't know if the stream is paused in stack trace now. 🥵